pgmpy/readwrite/ProbModelXML.py

```diff
@@ -483,32 +486,33 @@ def add_node(self, variable):
         variable_name = variable.attrib['name']
         self.probnet['Variables'][variable_name] = {}
         self.probnet['Variables'][variable_name]['type'] = variable.attrib['type']
-        self.probnet['Variables'][variable_name]['roles'] = variable.attrib['roles']
-        if variable.find('Comment'):
+        self.probnet['Variables'][variable_name]['role'] = variable.attrib['role']
+        if variable.find('Comment') is not None:
```

Comment:
If i use if variable.find('Comment') then while running tests it gives me following warning:

FutureWarning: The behavior of this method will change in future versions. Use specific 'len(elem)' or 'elem is not None' test instead.
  if variable.find('Comment'):